style.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  background: #191765;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
- [ ] You have done a great job using CSS flexbox to align your content, there is one thing you need to work on in order to make your CSS file even cleaner. I can see that you have used the property `display: flex` at multiple places, please let's use DRY(Don't repeat yourself) to keep the code clean by extracting the property into a separate class that can be reused at multiple places, see example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```